### DISCUSSION TITLE
How to set multicast input url with unicast source? (Source-specific multicast)

### DISCUSSION THREAD

**Discussion by Canta** [bluenviron/mediamtx#3957] [Questions]
### Question

I need to set up a _multicast URL with specific unicast source_ as a _mediamtx path source_.
Please note the difference in context for the "source" term. The second instance of that term points to mediamtx terminology, while the first one refers to [source-specific multicast, or SSM](https://en.wikipedia.org/wiki/Source-specific_multicast).

For example, I need to be able to do something like this:
```
paths:
  mystream:
    source: "udp://1.2.3.4@239.0.0.1:12345"
```
or this:
```
paths:
  mystream:
    source: "udp://239.0.0.1@1.2.3.4:12345"
```
or this:
```
paths:
  mystream:
    source: "udp://239.0.0.1:12345?source=1.2.3.4"
```

I haven't found a proper syntax to set this up, and I'm doubting if this is even possible.
Is there any way to use SSM with multicast UDP URLs as sources?

**Comment by aler9** [comment]
moved in #4178
